site/content/docs/0.0/migration.md

```diff
@@ -75,31 +75,112 @@ toc: true
 #### Shadow
 
 - <span class="badge text-bg-success">New</span> Shadows utilities:
-  `shadow-none`, `shadow-raised`, `shadow-drag`, `shadow-overlay-default`, `shadow-overlay-emphasized`, `shadow-sticky-default`, `shadow-sticky-emphasized` and `shadow-sticky-navigation-scrolled`.
+  `.shadow-none`, `.shadow-raised`, `.shadow-drag`, `.shadow-overlay-default`, `.shadow-overlay-emphasized`, `.shadow-sticky-default`, `.shadow-sticky-emphasized` and `.shadow-sticky-navigation-scrolled`.
 
 ### Examples
 
 - <span class="badge text-bg-success">New</span> New [Grid example]({{< docsref "/examples/grid" >}}).
 
 - <span class="badge text-bg-success">New</span> New [Grid system example]({{< docsref "/examples/grid-system" >}}).
 
+#### Spacings
```

Comment:
Spacing shouldn't be in the Utilities section rather than Example as it is now ?